docs/source/install.rst

```diff
@@ -6,7 +6,7 @@ Requirements
 
 * `NVIDIA CUDA GPU <https://developer.nvidia.com/cuda-gpus>`_ with the Compute Capability 3.0 or larger.
 
-* `CUDA Toolkit <https://developer.nvidia.com/cuda-toolkit>`_: v11.2 / v11.3 / v11.4 / v11.5 / v11.6 / v11.7 / v11.8 / v12.0 / v12.1 / v12.2
+* `CUDA Toolkit <https://developer.nvidia.com/cuda-toolkit>`_: v11.2 / v11.3 / v11.4 / v11.5 / v11.6 / v11.7 / v11.8 / v12.0 / v12.1 / v12.2 [#cuda112windows]_
```

Comment:
![image](https://github.com/cupy/cupy/assets/15824382/10c6a6b6-9218-461f-a04f-201c22714217)
This is the rendered doc, I just give it another thought and maybe it is better to specify (linux only)?
I think it is not very important but just a quick note :)

Reply:
The point is that CUDA 12.2 for Windows is totally fine for most users with CUDA Toolkit installed, except for environments without headers...

Reply:
It seems only supported footnote syntax is numbers (1, 2, 3, ...) or symbols (`*`, `✝️`, ...)
https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#footnotes

![image](https://github.com/cupy/cupy/assets/939877/6a59b0f2-6c46-4499-8a56-10d7e0616636)

Reply:
I guess it is fine, it can be seen at a glance so no prob!